jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     '\\.worker.entry.js': '<rootDir>/jestHelpers/mocks/workerMock.js'
   },
   transform: {
-    '^.+\\.(js|jsx)?$': '<rootDir>/babel-transformer.js',
```

Comment:
why `js` files are removed ?

Reply:
They are not removed /\.jsx?/ takes care of both js and jsx files.